app-editors/ghostwriter/ghostwriter-2.0.1.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils xdg-utils
+
+DESCRIPTION="Cross-platform, aesthetic, distraction-free markdown editor"
+HOMEPAGE="https://wereturtle.github.io/ghostwriter/"
+SRC_URI="https://github.com/wereturtle/ghostwriter/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+BDEPEND="dev-qt/linguist-tools:5"
+
+RDEPEND="
+	app-text/hunspell:=
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwebchannel:5
+	dev-qt/qtwebengine:5[widgets]
+	dev-qt/qtwidgets:5
+"
+DEPEND="${RDEPEND}"
+
+DOCS=( CREDITS.md README.md )
+
+PATCHES=(
+	"${FILESDIR}/${P}-appdata-path.patch"
+	"${FILESDIR}/${P}-qt-5.15.patch" # git master
```

Comment:
Did you forget to push these patches, or do the old ones apply?
```
>>> Preparing source in /var/tmp/portage/app-editors/ghostwriter-2.0.1/work/ghostwriter-2.0.1 ...
 * Applying ghostwriter-2.0.1-appdata-path.patch ...
/var/tmp/portage/app-editors/ghostwriter-2.0.1/temp/environment: line 400: /var/tmp/portage/app-editors/ghostwriter-2.0.1/files/ghostwriter-2.0.1-appdata-path.patch: No such file or directory
/var/tmp/portage/app-editors/ghostwriter-2.0.1/temp/environment: line 403: /var/tmp/portage/app-editors/ghostwriter-2.0.1/files/ghostwriter-2.0.1-appdata-path.patch: No such file or directory
 [ !! ]
 * ERROR: app-editors/ghostwriter-2.0.1::gentoo failed (prepare phase):
 *   patch -p1  failed with /var/tmp/portage/app-editors/ghostwriter-2.0.1/files/ghostwriter-2.0.1-appdata-path.patch
 * 
 * Call stack:
 *               ebuild.sh, line  127:  Called src_prepare
 *             environment, line 1198:  Called default
 *      phase-functions.sh, line  857:  Called default_src_prepare
 *      phase-functions.sh, line  922:  Called __eapi6_src_prepare
 *             environment, line  192:  Called eapply '/var/tmp/portage/app-editors/ghostwriter-2.0.1/files/ghostwriter-2.0.1-appdata-path.patch' '/var/tmp/portage/app-editors/ghostwriter-2.0.1/files/ghostwriter-2.0.1-qt-5.15.patch'
 *             environment, line  468:  Called _eapply_patch '/var/tmp/portage/app-editors/ghostwriter-2.0.1/files/ghostwriter-2.0.1-appdata-path.patch'
 *             environment, line  406:  Called __helpers_die 'patch -p1  failed with /var/tmp/portage/app-editors/ghostwriter-2.0.1/files/ghostwriter-2.0.1-appdata-path.patch'
 *   isolated-functions.sh, line  112:  Called die
 * The specific snippet of code:
 *   		die "$@"
```

Reply:
sry, patches aren't needed at all in the new version